core/src/com/unciv/models/Spy.kt

```diff
@@ -56,6 +56,8 @@ class Spy() : IsPartOfGameInfoSerialization {
     }
 
     fun endTurn() {
+        // If the city
+        if (getLocation() == null) moveTo(null)
```

Comment:
???

Reply:
It was kind of a hotfix for when a city was captured that the spy was in. I changed the behaviour to be when the city is captured instead of when the spy updates now. (Now that I read up on it, this is civ V behavior)